### ISSUE TITLE
Support for "json" extension

### ISSUE THREAD

**Issue by matteotolazzi** [atom-community/autocomplete-paths#155]
The paths autocompletion does not take into account .json files even if I configured the package as follows in config.cson:

`
 "autocomplete-paths":
    extensions: [
      "js"
      "jsx"
      "json"
    ]
`

**Comment by saschagehlich** [comment]
Fixed with 2.12.0

**Comment by bennypowers** [comment]
```cson
  "autocomplete-paths":
    enableHtmlSupport: true
    extensions: [ "js", "jsx", "json" ]
    ignoredPatterns: [
      "node_modules/**/node_modules/"
    ]
    maxFileCount: 2500
```

```
$ ls -1 test/test-data/
20m.0-2+2-5+10-12+12-15.json
20m.0-2+2-5+10-12+12-20.json
20m.1-2+2-5+10-12+12-15.json
20m.1-2+2-5+10-12+12-20.json
hi.html
hi.js
with-urls.json
```

<img width="507" alt="screen shot 2018-01-03 at 2 24 36 pm" src="https://user-images.githubusercontent.com/1466420/34520568-dbbd77ba-f091-11e7-954c-cf61c63335a5.png">

Version 2.12.1

Atom    : 1.23.2
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0

macOS 10.13.1
